react/features/app-settings/components/AbstractAppSettings.js

```diff
@@ -44,6 +42,8 @@ type Props = {
  * @abstract
  */
 export class AbstractAppSettings extends Component<Props> {
+    urlField: Object;
```

Comment:
Please move this to the native implementation, that's the only place where we use it.